src/main/java/nc/students/ayaz/deserializer/UserDeserializer.java

```diff
@@ -0,0 +1,28 @@
+package nc.students.ayaz.deserializer;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.ObjectCodec;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonNode;
+import nc.students.ayaz.model.User;
+import nc.students.ayaz.model.Video;
+
+import java.io.IOException;
+
+public class UserDeserializer extends JsonDeserializer<User> {
```

Comment:
&#x1F34E; **Issue**
Why there aren't any tests for deserializers ?
Its actually one of the easiest examples you cant think of to demonstrate TDD ;)